Documentation/contributing/development/hive.rst

```diff
@@ -780,6 +780,186 @@ Few guidelines one should strive to follow when implementing larger cells:
+    scripttest.go:253: $WORK=/tmp/TestScriptexample.txtar2477299450/001
+    scripttest.go:72:
+        DATADIR=/home/jussi/go/src/github.com/cilium/cilium/contrib/examples/script/testdata
+        PWD=/tmp/TestScriptexample.txtar2477299450/001
+        WORK=/tmp/TestScriptexample.txtar2477299450/001
+        TMPDIR=/tmp/TestScriptexample.txtar2477299450/001/tmp
+
+    scripttest.go:72: #! --enable-example=true
+        # ^ an (optional) shebang can be used to configure cells
+        # This is a comment that starts a section of commands (0.000s)
+        > echo 'hello'
+        [stdout]
+        hello
+    logger.go:256: level=INFO msg="Starting hive"
+    logger.go:256: level=INFO msg="Started hive" duration=1.53µs
+    scripttest.go:72: # The test hive has not been started yet, let's start it! (0.000s)
+        > hive/start
+    logger.go:256: level=INFO msg="SayHello() called" module=example name=foo greeting=Hello,
+    scripttest.go:72: # Cells can provide custom commands (0.000s)
+        > example/hello foo
+        calling SayHello(foo, Hello,)
+        [stdout]
+        Hello, foo
+        > stdout 'Hello, foo'
+        matched: Hello, foo
+    scripttest.go:72: # Check that call count equals 1 (0.000s)
+        > example/counts
+        [stdout]
+        1 SayHello()
+        > stdout '1 SayHello()'
+        matched: 1 SayHello()
+    scripttest.go:72: # The file 'foo' should not be the same as 'bar' (0.000s)
+        > ! cmp foo bar
+        diff foo bar
+        --- foo
+        +++ bar
+        @@ -1,2 +1,1 @@
+        -foo
+        -
+        +bar
+
+    --- PASS: TestScript/example.txtar (0.00s)
+    ok      github.com/cilium/cilium/contrib/examples/script        0.003s
+
+In the test execution we can see that a temporary working directory ``$WORK`` was created
+and our test files from the ``example.txtar`` extracted there. Each command was then executed
+in order.
+
+As many of the cells bring rich set of commands it's important that they're easy to discover.
+This is helped by having a help system in place (``help``) and the ability to interactively try things out
+with the ``break`` statement. Here's what would happen if you'd add ``break`` as the last command in ``example.txtar``:
+
+.. code-block:: shell-session
+
+  $ go test .
+    ....
+        @@ -1,2 +1,1 @@
+        -foo
+        -
+        +bar
+
+        > break
+
+  Break! Control-d to continue.
+  debug> help example
+  [stdout]
+  example/counts
+          Show the call counts of the example module
+  example/hello [--greeting=string] name
+          Say hello
+  
+          Flags:
+                --greeting string   Greeting to use (default "Hello,")
+
+  debug> example/hello --greeting=Hei Jussi
+  calling SayHello(Jussi, Hei)
+  [stdout]
+  Hei Jussi
+  logger.go:256: level=INFO msg="SayHello() called" module=example name=Jussi greeting=Hei
+
+Command reference
+^^^^^^^^^^^^^^^^^
+
+The important default commands are:
+
+- ``hive``: Dump the hive object graph
```

Comment:
nit: I see multiple passive references to `help`, but the text doesn't currently seem to actually instruct the reader to use this command. Maybe emphasizing in the command reference would be good enough:

```suggestion
The important default commands are:

- ``help``: List available commands
- ``hive``: Dump the hive object graph
```

Reply:
Added.